playbooks/webhooks.yml

```diff
@@ -35,6 +19,72 @@
       ufw:
         rule: allow
         port: 22
+        src: "{{ item }}"
+      with_items: "{{ ip_address_whitelist }}"
+
+    - name: Enable firewall
+      ufw:
+        state: enabled
+        policy: deny
+
+    - name: Add RPC public keys to authorized_keys
+      authorized_key:
+        user: "root"
+        key: "{{ lookup('file', lookup('env', 'WORKSPACE')+'/rpc-gating/keys/rcb.keys') }}"
+        state: "present"
+  roles:
+    - role: willshersystems.sshd
+      ListenAddress:
+        - "0.0.0.0"
+      Port: 22
+      Protocol: 2
+      HostKey:
+        - "/etc/ssh/ssh_host_rsa_key"
+        - "/etc/ssh/ssh_host_dsa_key"
+        - "/etc/ssh/ssh_host_ecdsa_key"
+        - "/etc/ssh/ssh_host_ed25519_key"
+      UsePrivilegeSeparation: "yes"
+      KeyRegenerationInterval: 3600
+      ServerKeyBits: 1024
+      SyslogFacility: "AUTH"
+      LogLevel: "INFO"
+      LoginGraceTime: 120
+      PermitRootLogin: "yes"
+      StrictModes: "yes"
+      RSAAuthentication: "yes"
+      PubkeyAuthentication: "yes"
+      IgnoreRhosts: "yes"
+      RhostsRSAAuthentication: "no"
+      HostbasedAuthentication: "no"
+      PermitEmptyPasswords: "no"
+      ChallengeResponseAuthentication: "no"
+      PasswordAuthentication: "no"
+      X11Forwarding: "no"
+      PrintMotd: "no"
+      PrintLastLog: "no"
+      TCPKeepAlive: "yes"
+      AcceptEnv: "LANG LC_*"
+      Subsystem: "sftp /usr/lib/openssh/sftp-server"
+      UsePAM: "yes"
+      GatewayPorts: "no"
+      ClientAliveInterval: 15
+      UseDNS: "no"
+      Compression: "yes"
+      CompressionLevel: 6
+    - role: jnv.unattended-upgrades
+      unattended_automatic_reboot_time: "00:00"
```

Comment:
I'd say don't enable automatic reboots and we handle reboots manually.  Keep in mind that this is only upgrading packages w/ security updates, so the # of updates should be quite minimal.